src/main/clojure/clara/rules.cljc

```diff
@@ -332,6 +332,12 @@
       * :forms-per-eval - The maximum number of expressions that will be evaluated per call to eval.
         Larger batch sizes should see better performance compared to smaller batch sizes. (Only applicable to Clojure)
         Defaults to 5000, see clara.rules.compiler/forms-per-eval-default for more information.
+      * :omit-compile-ctx - When false Clara, in Clojure, retains additional information to improve error messages during
```

Comment:
This should say "When true" right?

Also, why does it say ", in Clojure,"? What's the relevance of that?

Reply:
Ok, I guess it reads correctly after all. I guess I'm more used to flags described about what they do _when true_. So it's a bit weird that it starts of "when false", but it semantically is correct still.